flight_declaration_operations/migrations/0001_initial.py

```diff
@@ -17,7 +17,7 @@ class Migration(migrations.Migration):
             name='FlightDeclaration',
             fields=[
                 ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
-                ('operational_intent', models.TextField()),
+                ('operational_intent', models.JSONField()),
```

Comment:
Changed the type to JSON so that we can query the altitude values via Django

Reply:
I guess from spec point of view operational_intent in JSON sud be ok(REF: UAS-Flight-Authorisation-Automated-Testing.pdf)